qa/suites/ceph-ansible/smoke/basic/4-tasks/rbd_import_export.yaml

```diff
@@ -3,5 +3,5 @@ meta:
 tasks:
 - workunit:
     clients:
-      client.0:
+      installer.0:
```

Comment:
No need for this change as commented earlier.

Reply:
how does`ceph-ansible` know to run on `installer.0` ?